vocabularies/ODM.xml

```diff
@@ -33,17 +33,35 @@
         </Collection>
       </Annotation>
 
-      <Term Name="codeList" Type="Core.Tag" DefaultValue="true" AppliesTo="EntityType EntitySet">
+      <Term Name="codeList" Type="Core.Tag" Nullable="false" DefaultValue="true" AppliesTo="EntityType EntitySet">
         <Annotation Term="Common.Experimental" />
         <Annotation Term="Core.Description" String="The annotated entity set represents a list of code values, the annotated entity type represents an entry in a code list" />
       </Term>
 
-      <Term Name="root" Type="Core.Tag" DefaultValue="true" AppliesTo="EntitySet">
+      <Term Name="root" Type="Core.Tag" Nullable="false" DefaultValue="true" AppliesTo="EntitySet">
         <Annotation Term="Common.Experimental" />
         <Annotation Term="Core.Description" String="The annotated entity set contains root entities" />
         <Annotation Term="Core.LongDescription" String="Root Entities have an independent lifetime and are reachable as top level resources in APIs. These are usually the key entities of a domain, sometimes also called 'business objects'. They are called aggregate roots in DDD terminology." />
       </Term>
 
+      <Term Name="oid" Type="Edm.PropertyPath" Nullable="false" AppliesTo="EntityType">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="The named field is an OID (ODM identifier), and it uniquely identifies an ODM root entity in a customer landscape." />
+        <Annotation Term="Core.LongDescription">
+          <String>The technical type can be UUID or String (any length), and the value must be stable and unique at least within type and landscape (better globally). 
+An entity may have other IDs, i.e. a local ID and other alternative IDs. This is the ID used for ODM.</String>
```

Comment:
```suggestion
An entity may have other IDs, i.e. a local ID and other alternative IDs. </String>
```
"This is the ID used for ODM" should be obvious from being an ODM ID. It doesn't add more to it, does it? Therefore I would remove it.